### ISSUE TITLE
Error in queue runner: "Could not find EventbriteLink 'Event' record for order X with 'event_id': Y; cannot process Order.";

### ISSUE THREAD

**Issue by twomice** [twomice/com.joineryhq.eventbrite#1]
@josephlacey and I were surprised today to get this and several other similar error messages in the queue runner:

> "Could not find EventbriteLink 'Event' record for order 959231762 with 'event_id': 57234242159; cannot process Order.";

Joseph had just purchased a ticket for on event (which we'll call "Testing Copy Event"), which itself was copied from the main event, which we'll call "Main Live Event". We were expecting to see three entries in the queue table from that purchase: one order.updated and two attendee.updated.  We also expected that if we ran the queue runner job, those queue entries would be found and processed.

Instead we found that no queue entries were successfully processed, and several entries encountered this error or an equivalent one (i.e., with different values for orderId and eventId).

Event ID for "Testing Copy Event" is: 63000711825
Event ID for "Main Live Event" is: 57234242159

**Question: Why after purchasing a ticket for "Testing Copy Event" (ID = 63000711825) are we getting errors for an order that says it's for "Main Live Event" (ID = 57234242159)?**

**Comment by twomice** [comment]
Looking into this, I'm finding a perfectly reasonable "not a bug" explanation, except perhaps it's a bug to report this situation as an error. Here's what I found:

- Order 959231762 is indeed for "Main Live Event" (ID = 57234242159) (reference: https://www.eventbriteapi.com/v3/orders/959231762/?token=[YOUR_TOKEN].  It was created at "2019-05-29T20:12:18Z" and updated today, at "2019-06-05T16:03:21Z".
- The status of this order is 'deleted'. Apparently it was deleted today at the 'updated' date/time.
- This 'updated' date/time is after we established the order.update webhook; this explains why the order appears in the queue table.

Instead of reporting an error, correct behavior here would be:
1. simply to ignore the order, because it's for an event which is not configured for integration.
2. mark the queue entry as processed and report it as such in the queue runner job log.
3. move on.

TODO: Implement the above change.
